app/controllers/cinema_halls_controller.rb

```diff
@@ -4,12 +4,12 @@ class CinemaHallsController < ApplicationController
   include JSONAPI::Fetching
 
   def index
-    @cinema_halls = CinemaHall.all
+    @cinema_halls = CinemaHalls::Representers::Multiple.new.call
```

Comment:
You should pass to the initializer collection of cinema_halls. For example:
```
cinema_halls = CinemaHall.all
CinemaHalls::Representers::Multiple.new(cinema_halls).call
```
Please check do you still need jsonapi if you have a representer.